apps/analysis/tasks.py

```diff
@@ -89,7 +89,7 @@ def run_analysis(run_group_id: int):
     elif group.is_complete:
         return
 
-    with RunStatusContext(group):
+    with RunStatusContext(group, bubble_errors=False):
```

Comment:
Why this change?

Reply:
This is the outermost context manager so if it bubbles errors it will cause the celery task to fail which isn't what we want.